### PR TITLE
Add support for GHC 9.0.2

### PR DESCRIPTION
**Pull request by nitinprakash96**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -11,21 +11,14 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest, macOS-latest, windows-latest]
-        cabal: ["3.4"]
+        os:
+          - ubuntu-latest
+        cabal:
+          - 3.4
```
Comment (jhrcek) [inline .github/workflows/ci.yml:17]: I'd change this to 3.6, just to be on the latest recommended version
```suggestion
          - 3.6
```

File: fcm-client.cabal [hunk 2]
```diff
@@ -18,6 +18,7 @@ tested-with:         GHC == 8.4.4
                      GHC == 8.8.4
                      GHC == 8.10.4
                      GHC == 8.10.7
+                     GHC == 9.0.2
```
Comment (jhrcek) [inline fcm-client.cabal:21]: I'd remove the 8.10.4 from the "tested-with" now that we're dropping that from CI

**Review by jhrcek — COMMENTED** [review]
Looks OK, just two minor suggestions.

File: fcm-client.cabal (hunk 2 above)
Comment (nitinprakash96) [reply fcm-client.cabal:18]: Done.

**Review by jhrcek — APPROVED** [review]
LGTM

**Pull request merged**